cogstat/cogstat_stat.py

```diff
@@ -699,6 +699,168 @@ def variable_pair_standard_effect_size(data, meas_lev, sample=True, normality=No
                                                                                  classes="table_cs_pd"))
     return standardized_effect_size_result
 
+def calc_se_r2(r2, k, n):
```

Comment:
this function should go to the cogstat_stat_num module

Reply:
Functions moved.